scarb/src/bin/scarb/commands/new.rs

```diff
@@ -18,7 +18,7 @@ pub fn run(args: NewArgs, config: &Config) -> Result<()> {
             } else {
                 VersionControl::Git
             },
-            snforge: args.init.snforge,
+            snforge: matches!(args.init.test_runner, Some(TestRunner::StarknetFoundry)),
```

Comment:
I feel like we could run the prompt here, instead of modyfing the command at `bin/main.rs`.